{{cookiecutter.repo_name}}/Procfile.dev

```diff
@@ -1,3 +1,3 @@
 web: PYTHONUNBUFFERED=True python manage.py runserver 0.0.0.0:$PORT
 static: npm run dev
-rqworker: PYTHONUNBUFFERED=True python manage.py rqworkers high default low --autoreload
```

Comment:
oh no! you removed django-rqworker. I suppose it probably doesn't work with django 4.0 (I'm not sure it worked with 2 or 3 either)

Reply:
@joesolly  I forgot I had this PR open. I really was just updating my repo.  Don't want to merge this in repo.... at least not yet.
But yeah RQ worker stopped working in 3.0 I think. Maybe we could get a few people together to see if there is any interests in fixing and maintaining. I know we talked about once.

Reply:
@CorneliusIV Maybe, unfortunately, I currently only work in celery or RMQ, so I don't have any work motivation to fix this. Hackathon? Pizza and Rosé?